addons/mail/static/src/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -21,6 +21,11 @@
 
 .o_ChatterTopbar_tools {
     padding-right: var(--ChatterTopbar-padding-right, 0);
+    border-bottom: var(--ChatterTopbar-border);
+
+    &_border, & .o_ChatterTopbar_center {
```

Comment:
In general not a good idea to build a full classname with partial `&`, because when you grep the classname from template you see no match, so we'll likely assume it's unused and just remove the classname in template.

Reply:
Hello, thanks for your review, I changed the selector with the full classname